### PR TITLE
uploading UserStory5

### PR DESCRIPTION
**Pull request by NavidaritoChen-Ksquare**

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -41,3 +41,8 @@ button {
   color: white;
 }
 
+.counter{
+  font-size: 50px;
+
```
Comment (MarianaLopez-Ksquare) [inline index.css]: Delete extra spaces pls

File: index.js [hunk 2]
```diff
@@ -233,6 +238,9 @@ class GameOfLife {
         this.currentGeneration[row][col] = 0;
       }
     }
+    this.count = 0;
+    document.querySelector('.counter').innerHTML = "";
+
```
Comment (MarianaLopez-Ksquare) [inline index.js]: Same, delete extra spaces

**Review by MarianaLopez-Ksquare — CHANGES_REQUESTED** [review]
Looks great, just delete extra spaces

File: index.js [hunk 3]
```diff
@@ -100,6 +102,9 @@ class GameOfLife {
       this.nextGeneration = await this.generateNextGeneration();
 
       this.currentGeneration = this.nextGeneration;
+      console.log(this.currentGeneration)
```
Comment (MarianaLopez-Ksquare) [inline index.js]: delete this please

**Review by MarianaLopez-Ksquare — CHANGES_REQUESTED** [review]

**Review by EstrellaPoot-Ksquare — APPROVED** [review]

**Review by MarianaLopez-Ksquare — APPROVED** [review]

**Pull request merged**